pkg/networking/cilium/reconciler/reconciler.go

```diff
@@ -260,3 +279,25 @@ func (r *Reconciler) installPreflight(ctx context.Context, client client.Client,
 
 	return nil
 }
+
+func (r *Reconciler) reconcileSpecialCases(ctx context.Context, c client.Client, yaml []byte) ([]client.Object, error) {
```

Comment:
Presumable this needs removing in a subsequent release: have we an issue to track?

Reply:
No, we don't have one yet but I can create one for us to track

Reply:
Thanks, Taylor. Probably stick a blocker label on it straight away so its clear we need it.

Reply:
For sure! Added the label